### ISSUE TITLE
$t says No key found if globalInjection is set to true

### ISSUE THREAD

**Issue by tomocrafter** [intlify/bundle-tools#20]
**Reporting a bug?**
If I set the `legacy: false` and `globalInjection: true` in createI18n, the local defined messages with i18n tag in SFC(Single File Component) makes $t with it returns key and occurs warning.

**Module versions (please complete the following information):**
 - `@vue/cli`: 4.5.12
 - `vue-i18n`: 9.1.4
 - `vue-cli-plugin-i18n`: 2.1.0
 - `vue`: 3.0.5

**To Reproduce**
1. Set `legacy: false` as below
```TypeScript
const i18n = createI18n({
  legacy: false,
  globalInjection: true,
  locale: 'ja',
  fallbackLocale: 'ja',
  fallbackWarn: false,
  messages: {
    ja
  }
})

app.use(i18n)
```
2. Create local i18n with i18n tag in SFC
3. Use $t in the template
4. See the warning that says there is no key in locale that defined in i18n tag.

**Expected behavior**
$t returns translated string as expected

**Comment by PeterAlfredLee** [comment]
@angusryer Thank you for reporting this. I think the `.default` problem It's been fixed since `vue-cli-plugin-i18n@v2.1.2`. 
See also [vue-i18n-next issue #597](https://github.com/intlify/vue-i18n-next/issues/597) and [vue-cli-plugin-i18n pr #217](https://github.com/intlify/vue-cli-plugin-i18n/pull/217).

**Comment by PeterAlfredLee** [comment]
Hi @tomocrafter, Is it everything works fine now? Can you close this issue?
